nanobind_stubgen/NanobindStubsGenerator.py

```diff
@@ -76,7 +76,7 @@ def export(self, output_path: Path, intent: int = 0):
             module_path = output_path.joinpath(f"{self.name}.pyi")
 
         # create init file
-        out = [f"from typing import Any, Optional, overload, Typing, Sequence",
+        out = [f"from typing import Any, Optional, overload, Typing, Sequence, Iterable, Union, Callable",
```

Comment:
Somewhat unrelated to the PR, but: I think the `Typing` import here ought to be removed? The symbol does not exist in `typing` (at least for Python 3.11; I haven't tested other versions) and appears to be unused.

Reply:
I agree, we need to fix imports differently (not just adding more and more).